pkg/apis/serving/v1alpha1/kfservice_validation.go

```diff
@@ -90,6 +93,12 @@ func validateDefaultSpec(defaultSpec ModelSpec) error {
 	if err := validateOneModelSpec(defaultSpec); err != nil {
 		return err
 	}
+	if defaultSpec.Custom != nil {
```

Comment:
should we push the nil check into the validator?

Reply:
done